updates for authenticator (#3849)
aws-amplify/docs: src/fragments/lib/auth/flutter/getting_started/30_initAuth.mdx

Your code should look like this:

```dart
import 'package:amplify_flutter/amplify.dart';
import 'package:amplify_auth_cognito/amplify_auth_cognito.dart';
import 'package:flutter/material.dart';

import 'amplifyconfiguration.dart';

void main() {
  runApp(MyApp());
}

class MyApp extends StatefulWidget {
  @override
  _MyAppState createState() => _MyAppState();
}

class _MyAppState extends State<MyApp> {
  @override
  void initState() {
    super.initState();
    _configureAmplify();
  }

  void _configureAmplify() async {
    // Add the following line to add Auth plugin to your app.
    Amplify.addPlugins(AmplifyAuthCognito()]);

    try {
      // call Amplify.configure to use the initialized categories in your app
      await Amplify.configure(amplifyconfig);
    } on AmplifyAlreadyConfiguredException {
      print(
          "Tried to reconfigure Amplify; this can occur when your app restarts on Android.");
    }
  }
  
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
        home: Scaffold(

>             body: Container()

Author: Addressed this now!

>   }

Author: suggestions committed

Add the Auth plugin, along with any other plugins you may have added as described in the **Prerequisites** section. To complete initialization call `Amplify.configure()`.

Your code should look like this:

```dart
import 'package:amplify_flutter/amplify.dart';
import 'package:amplify_auth_cognito/amplify_auth_cognito.dart';
import 'package:flutter/material.dart';

import 'amplifyconfiguration.dart';

void main() {
  runApp(MyApp());
}

> class MyApp extends StatefulWidget {
>   @override
>   _MyAppState createState() => _MyAppState();
> }

Reviewer: ```suggestion
class MyApp extends StatefulWidget {
  const MyApp({Key? key}): super(key: key);

  @override
  _MyAppState createState() => _MyAppState();
}
```

class _MyAppState extends State<MyApp> {
  @override
  void initState() {
    super.initState();
    _configureAmplify();
  }

>   void _configureAmplify() async {
>     // Add the following line to add Auth plugin to your app.
>     Amplify.addPlugins(AmplifyAuthCognito()]);
> 
>     try {
>       // call Amplify.configure to use the initialized categories in your app
>       await Amplify.configure(amplifyconfig);
>     } on AmplifyAlreadyConfiguredException {
>       print(
>           "Tried to reconfigure Amplify; this can occur when your app restarts on Android.");
>     }
>   }

Reviewer: ```suggestion
  Future<void> _configureAmplify() async {
    try {
      // Add the following line to add Auth plugin to your app.
      await Amplify.addPlugin(AmplifyAuthCognito());

      // call Amplify.configure to use the initialized categories in your app
      await Amplify.configure(amplifyconfig);
    } on Exception catch (e) {
      print('An error occurred configuring Amplify: $e');
    }
  }
```

>   @override
>   Widget build(BuildContext context) {
>     return MaterialApp(
>         home: Scaffold(
>             body: Container(),
>             ),
>             );
>   }

Reviewer: ```suggestion
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      home: Scaffold(
        body: Container(),
      ),
    );
  }
```
